readme.md

```diff
@@ -183,6 +184,12 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [A Deep Dive into eBPF: Writing an Efficient DNS Monitoring.](https://medium.com/@nurkholish.halim/a-deep-dive-into-ebpf-writing-an-efficient-dns-monitoring-2c9dea92abdf) - A detailed explanation of methods used to capture DNS requests at the socket filter layer.
 - [eBPF Developer Tutorial - Learn eBPF by examples](https://eunomia.dev/tutorials/) - Start with eBPF basics and progress to advanced topics using 20+ hands-on tutorials and examples. Covers performance, networking, and security with libbpf and CO-RE. Available in Chinese and English.
 - [Catch Performance Regressions in eBPF](https://bencher.dev/docs/explanation/talks/#linuxcon-2023-12-may-23) - A step-by-step guide to benchmarking both the client and kernel eBPF code written in Rust.
+- [Loops and Iterators in eBPF](https://cloudchirp.substack.com/p/loops-and-iterators-in-ebpf?r=1m86sa) - Newsletter about all the ways to loop and iterate in eBPF.
```

Comment:
Hi, thanks and apologies for the delay.

The contents look good (nice work!), so I'm fine with adding the links. That's a lot of links, but then that's also a lot of interesting reads.

Could you please clean up the links? Some seem to have a parameter that is not strictly necessary to access the resource:

```suggestion
- [Loops and Iterators in eBPF](https://cloudchirp.substack.com/p/loops-and-iterators-in-ebpf) - Newsletter about all the ways to loop and iterate in eBPF.
```

Reply:
Oh, I see - this should now be resolved.